From 9eb6762e35b14bd207fa5d7359bd692759b71bfb Mon Sep 17 00:00:00 2001
From: Donald Sharp <sharpd@nvidia.com>
Date: Wed, 21 Apr 2021 08:57:29 -0400
Subject: [PATCH 3/5] tools: Fix warning when running frr-reload.py

When I run frr-reload.py I am seeing this error:

Apr 21 06:23:51 eva frrinit.sh[3776992]: /usr/lib/frr/frr-reload.py:1094: SyntaxWarning: "is not" with a literal. Did you mean "!="?
Apr 21 06:23:51 eva frrinit.sh[3776992]:   if line is not "exit-vrf":

fix

Signed-off-by: Donald Sharp <sharpd@nvidia.com>
---
 tools/frr-reload.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/frr-reload.py b/tools/frr-reload.py
index 88a40d89e..4a5ac8a1b 100755
--- a/tools/frr-reload.py
+++ b/tools/frr-reload.py
@@ -802,7 +802,7 @@ def check_for_exit_vrf(lines_to_add, lines_to_del):
                 add_exit_vrf = False
 
         if ctx_keys[0].startswith('vrf') and line:
-            if line is not "exit-vrf":
+            if line != "exit-vrf":
                 add_exit_vrf = True
                 prior_ctx_key = (ctx_keys[0])
             else:
-- 
2.17.1

